### PR TITLE
Rearrange/reword storage release notes for 1.6

### PR DESCRIPTION
**Pull request by saad-ali**
CC @kubernetes/sig-storage-misc

### CODE REVIEW

File: release-1.6/release-notes-draft.md [hunk 1]
```diff
@@ -171,18 +171,21 @@ The features described above are now specified using fields rather than annotati
 
 ## Storage
 ### New Features
+- Volume plugin for ScaleIO enabling pods to seamlessly access and use data stored on Dell EMC ScaleIO volumes.
+- Volume plugin for Portworx added capability to use [Portworx](http://www.portworx.com) as a storage provider for Kubernetes clusters. Portworx pools server capacity and turns servers or cloud instances into converged, highly available compute and storage nodes.
+- Volume plugin capable of projecting secrets, configmaps, and downward API items into the same directory.
+- Added support for mount options in persistent volumes.
+- Added ability to populate environment variables from a configmap or secret.
+- Add support to use NFSv3, NFSv4, and GlusterFS on GCE/GKE GCI image based clusters.
+- Support for user-written/run dynamic PV provisioners. See github.com/kubernetes-incubator/external-storage for a golang library and examples.
```
Comment (jeffvance) [inline release-1.6/release-notes-draft.md]: can the url be a link?

**Comment by calebamiles** [comment]
using a link now, @jeffvance

**Comment by calebamiles** [comment]
@saad-ali, @jeffvance could you or someone from @kubernetes/sig-storage-misc please collect, summarize, and remove the autogenerated release notes from the [bottom of the doc](https://github.com/kubernetes/features/blob/03d42c26ca586f2e13fdf8b2a72154ab1bbb1d1e/release-1.6/release-notes-draft.md#auto-generated-release-notes). Thanks!

**Comment by jeffvance** [comment]
@calebamiles Under Flex Volume Plugin there is this "Breaking changes": _Flex volume plugin is updated to support attach/detach interfaces_, however this change does not appear to be listed in the "Action Required" summary.

**Comment by jeffvance** [comment]
Maybe for a future pr, but it would be nice if there was a way to generate an index or TOC based on important words, such has "taints", "volumes", "storage class" etc. Maybe there are too many "important" words for this be useful, but, thinking out loud, it would nice to see some order to the action-able and noteworthy changes.

**Comment by saad-ali** [comment]
@ethernetdan can we merge this please

**Comment by idvoretskyi** [comment]
@saad-ali I will include your updates to my PR with the pre-final release notes.

**Pull request merged**